massa-execution/src/worker.rs

```diff
@@ -243,7 +241,7 @@ impl ExecutionWorker {
             while self.last_active_slot < end_step {
                 self.last_active_slot =
```

Comment:
I would prefer to assign to a variable like `next_slot`, with a comment `slot S`, followed by the call to `push_request` commented as `call the VM to execute an SCE-active miss at slot S`, and then assign `next_slot` to `self.last_active_slot` with a comment `set last_active_slot = S`, so that the code matches the spec more clearly. Ideally we should do this kind of commenting/matching of the spec everywhere.

Reply:
done